src/styles/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 166px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  color: var(--main-accent);
+  align-items: center;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__product-info {
+  margin-bottom: 16px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  margin-bottom: 4px;
+  line-height: 18px;
+}
+
+.card__product-code {
+  color: var(--secondary-accent);
+  font-size: 10px;
+}
+
+.card__review {
+  display: flex;
+  width: 100%;
+  margin-bottom: 24px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.review_value {
+  font-size: 10px;
+}
+
+.card__price {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-label {
+  font-size: 12px;
+  color: var(--secondary-accent);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__button-link {
+  width: 100%;
+}
+
+.card__button {
+  background-color: #00acdc;
+  width: 100%;
+  border-radius: 5px;
+  border-style: none;
+  border: 1px solid #00acdc;
+  color: white;
+  font-weight: 700;
+  font-size: 14px;
+  padding: 10px 0 12px;
```

Comment:
Why not `11px 0`? The text is not centered this way 🤔

Reply:
Yesterday it didn't pass the tests when I gave padding: 11px 0, but for some reason it passed now, so i fixed it.